pkg/rpc/adminservice/base.go

```diff
@@ -98,11 +98,18 @@ func NewAdminServer(kubeConfig, master string) *AdminService {
 	publisher := notifications.NewNotificationsPublisher(*configuration.ApplicationConfiguration().GetNotificationsConfig(), adminScope)
 	processor := notifications.NewNotificationsProcessor(*configuration.ApplicationConfiguration().GetNotificationsConfig(), adminScope)
 	go func() {
+		logger.Info(context.Background(), "Started processing notifications.")
 		err = processor.StartProcessing()
```

Comment:
shouldnt start processing auto-handle connection loss and reconnect failures? Ofcourse it should log

Reply:
sure, refactored @kumare3